lib/tether/ops_linux.go

```diff
@@ -393,6 +399,21 @@ func updateRoutes(t Netlink, link netlink.Link, endpoint *NetworkEndpoint) error
 	return nil
 }
 
+func bridgeTableExists(t Netlink) bool {
```

Comment:
It is required here so that we force redirect traffic that would normally be routed to one interface (management in this case), to another (public). A custom routing table seems to be the standard way to do this.